{
	"document": {
		"aggregate_severity": {
			"namespace": "https://access.redhat.com/security/updates/classification/",
			"text": "Moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright Â© Red Hat, Inc. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "An update for python-django is now available for Red Hat OpenStack Platform\n17.1 (Wallaby).\n\nRed Hat Product Security has rated this update as having a security impact\nof Moderate. A Common Vulnerability Scoring System (CVSS) base score, which\ngives a detailed severity rating, is available for each vulnerability from\nthe CVE link(s) in the References section.",
				"title": "Topic"
			},
			{
				"category": "general",
				"text": "Django is a high-level Python Web framework that encourages rapid\ndevelopment and a clean, pragmatic design. It focuses on automating as much\nas possible and adhering to the DRY (Don't Repeat Yourself) principle.\n\nSecurity Fix(es):\n\n* denial-of-service in ``intcomma`` template filter (CVE-2024-24680)\n\nFor more details about the security issue(s), including the impact, a CVSS\nscore, acknowledgments, and other related information, refer to the CVE\npage listed in the References section.",
				"title": "Details"
			},
			{
				"category": "legal_disclaimer",
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://access.redhat.com/security/team/contact/",
			"issuing_authority": "Red Hat Product Security is responsible for vulnerability handling across all Red Hat products and services.",
			"name": "Red Hat Product Security",
			"namespace": "https://www.redhat.com"
		},
		"references": [
			{
				"category": "self",
				"summary": "https://access.redhat.com/errata/RHSA-2024:2731",
				"url": "https://access.redhat.com/errata/RHSA-2024:2731"
			},
			{
				"category": "external",
				"summary": "https://access.redhat.com/security/updates/classification/#moderate",
				"url": "https://access.redhat.com/security/updates/classification/#moderate"
			},
			{
				"category": "external",
				"summary": "2261856",
				"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2261856"
			},
			{
				"category": "self",
				"summary": "Canonical URL",
				"url": "https://security.access.redhat.com/data/csaf/v2/advisories/2024/rhsa-2024_2731.json"
			}
		],
		"title": "Red Hat Security Advisory: Red Hat OpenStack Platform 17.1 (python-django) security update",
		"tracking": {
			"current_release_date": "2024-11-06T05:47:33+00:00",
			"generator": {
				"date": "2024-11-06T05:47:33+00:00",
				"engine": {
					"name": "Red Hat SDEngine",
					"version": "4.1.1"
				}
			},
			"id": "RHSA-2024:2731",
			"initial_release_date": "2024-05-22T20:35:57+00:00",
			"revision_history": [
				{
					"date": "2024-05-22T20:35:57+00:00",
					"number": "1",
					"summary": "Initial version"
				},
				{
					"date": "2024-05-22T20:35:57+00:00",
					"number": "2",
					"summary": "Last updated version"
				},
				{
					"date": "2024-11-06T05:47:33+00:00",
					"number": "3",
					"summary": "Last generated version"
				}
			],
			"status": "final",
			"version": "3"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "Red Hat OpenStack Platform 17.1",
								"product": {
									"name": "Red Hat OpenStack Platform 17.1",
									"product_id": "9Base-RHOS-17.1",
									"product_identification_helper": {
										"cpe": "cpe:/a:redhat:openstack:17.1::el9"
									}
								}
							}
						],
						"category": "product_family",
						"name": "Red Hat OpenStack Platform"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-django-0:2.2.24-8.el9ost.src",
								"product": {
									"name": "python-django-0:2.2.24-8.el9ost.src",
									"product_id": "python-django-0:2.2.24-8.el9ost.src",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python-django@2.2.24-8.el9ost?arch=src"
									}
								}
							}
						],
						"category": "architecture",
						"name": "src"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python3-django-0:2.2.24-8.el9ost.noarch",
								"product": {
									"name": "python3-django-0:2.2.24-8.el9ost.noarch",
									"product_id": "python3-django-0:2.2.24-8.el9ost.noarch",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python3-django@2.2.24-8.el9ost?arch=noarch"
									}
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					}
				],
				"category": "vendor",
				"name": "Red Hat"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-django-0:2.2.24-8.el9ost.src as a component of Red Hat OpenStack Platform 17.1",
					"product_id": "9Base-RHOS-17.1:python-django-0:2.2.24-8.el9ost.src"
				},
				"product_reference": "python-django-0:2.2.24-8.el9ost.src",
				"relates_to_product_reference": "9Base-RHOS-17.1"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-django-0:2.2.24-8.el9ost.noarch as a component of Red Hat OpenStack Platform 17.1",
					"product_id": "9Base-RHOS-17.1:python3-django-0:2.2.24-8.el9ost.noarch"
				},
				"product_reference": "python3-django-0:2.2.24-8.el9ost.noarch",
				"relates_to_product_reference": "9Base-RHOS-17.1"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2024-24680",
			"discovery_date": "2024-01-30T00:00:00+00:00",
			"ids": [
				{
					"system_name": "Red Hat Bugzilla ID",
					"text": "2261856"
				}
			],
			"notes": [
				{
					"category": "description",
					"text": "A vulnerability was found in Django. When used with very long strings, the intcomma template filter was subject to a potential denial of service attack.",
					"title": "Vulnerability description"
				},
				{
					"category": "summary",
					"text": "Django: denial-of-service in ``intcomma`` template filter",
					"title": "Vulnerability summary"
				},
				{
					"category": "general",
					"text": "The CVSS score(s) listed for this vulnerability do not reflect the associated product's status, and are included for informational purposes to better understand the severity of this vulnerability.",
					"title": "CVSS score applicability"
				}
			],
			"product_status": {
				"fixed": [
					"9Base-RHOS-17.1:python-django-0:2.2.24-8.el9ost.src",
					"9Base-RHOS-17.1:python3-django-0:2.2.24-8.el9ost.noarch"
				]
			},
			"references": [
				{
					"category": "self",
					"summary": "Canonical URL",
					"url": "https://access.redhat.com/security/cve/CVE-2024-24680"
				},
				{
					"category": "external",
					"summary": "RHBZ#2261856",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2261856"
				},
				{
					"category": "external",
					"summary": "https://www.cve.org/CVERecord?id=CVE-2024-24680",
					"url": "https://www.cve.org/CVERecord?id=CVE-2024-24680"
				},
				{
					"category": "external",
					"summary": "https://nvd.nist.gov/vuln/detail/CVE-2024-24680",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2024-24680"
				},
				{
					"category": "external",
					"summary": "https://github.com/advisories/GHSA-xxj9-f6rv-m3x4",
					"url": "https://github.com/advisories/GHSA-xxj9-f6rv-m3x4"
				},
				{
					"category": "external",
					"summary": "https://www.djangoproject.com/weblog/2024/feb/06/security-releases/",
					"url": "https://www.djangoproject.com/weblog/2024/feb/06/security-releases/"
				}
			],
			"release_date": "2024-02-06T15:00:00+00:00",
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "For details on how to apply this update, which includes the changes\ndescribed in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258",
					"product_ids": [
						"9Base-RHOS-17.1:python-django-0:2.2.24-8.el9ost.src",
						"9Base-RHOS-17.1:python3-django-0:2.2.24-8.el9ost.noarch"
					],
					"restart_required": {
						"category": "none"
					},
					"url": "https://access.redhat.com/errata/RHSA-2024:2731"
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"attackComplexity": "LOW",
						"attackVector": "NETWORK",
						"availabilityImpact": "HIGH",
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"confidentialityImpact": "NONE",
						"integrityImpact": "NONE",
						"privilegesRequired": "NONE",
						"scope": "UNCHANGED",
						"userInteraction": "NONE",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"9Base-RHOS-17.1:python-django-0:2.2.24-8.el9ost.src",
						"9Base-RHOS-17.1:python3-django-0:2.2.24-8.el9ost.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"details": "Moderate"
				}
			],
			"title": "Django: denial-of-service in ``intcomma`` template filter"
		}
	]
}
